React/Views/RCTViewManager.m

```diff
@@ -14,6 +14,7 @@
 #import "RCTEventDispatcher.h"
 #import "RCTLog.h"
 #import "RCTShadowView.h"
+#import "RCTSinglelineTextInputView.h"
```

Comment:
Let's tag this with our GH issue as well (and maybe a quick blurb like "needed for key-view loop special casing")